src/Workspaces/Core/Portable/Workspace/Solution/TextDocumentState.cs

```diff
@@ -77,20 +77,8 @@ private static ITextAndVersionSource CreateStrongText(TextAndVersion text)
         private static ITextAndVersionSource CreateStrongText(TextLoader loader)
             => new LoadableTextAndVersionSource(loader, cacheResult: true);
 
-        private static ITextAndVersionSource CreateRecoverableText(TextAndVersion text, LoadTextOptions loadTextOptions, SolutionServices services)
-        {
-            var result = new RecoverableTextAndVersion(new ConstantTextAndVersionSource(text), services);
-
-            // This RecoverableTextAndVersion is created directly from a TextAndVersion instance. In its initial state,
-            // the RecoverableTextAndVersion keeps a strong reference to the initial TextAndVersion, and only
-            // transitions to a weak reference backed by temporary storage after the first time GetValue (or
-            // GetValueAsync) is called. Since we know we are creating a RecoverableTextAndVersion for the purpose of
-            // avoiding problematic address space overhead, we call GetValue immediately to force the object to weakly
-            // hold its data from the start.
-            result.GetValue(loadTextOptions, CancellationToken.None);
-
-            return result;
-        }
+        private static ITextAndVersionSource CreateRecoverableText(TextAndVersion text, SolutionServices services)
+            => new RecoverableTextAndVersion(new ConstantTextAndVersionSource(text), services);
```

Comment:
```suggestion
        private static ITextAndVersionSource CreateRecoverableText(TextAndVersion text, LoadTextOptions loadTextOptions, SolutionServices services)
        {
            var result = new RecoverableTextAndVersion(new ConstantTextAndVersionSource(text), services);

            // This RecoverableTextAndVersion is created directly from a TextAndVersion instance. In its initial state,
            // the RecoverableTextAndVersion keeps a strong reference to the initial TextAndVersion, and only
            // transitions to a weak reference backed by temporary storage after the first time GetValue (or
            // GetValueAsync) is called. Since we know we are creating a RecoverableTextAndVersion for the purpose of
            // avoiding problematic address space overhead, we call GetValue immediately to force the object to weakly
            // hold its data from the start.
            result.GetValue(loadTextOptions, CancellationToken.None);

            return result;
        }
```